<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WAHEGURU" failures="1" tests="1" name="validate Test" time="0.181" errors="0" timestamp="2023-04-30T03:12:49 IST">
  <testcase classname="testpractice.ValidateSoftAssert" name="valiadte" time="0.031">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [jih] but found [Avg],
	custom error message did not expect to find [true] but found [false],
	intentional fail message">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [jih] but found [Avg],
custom error message did not expect to find [true] but found [false],
intentional fail message
at testpractice.ValidateSoftAssert.valiadte(ValidateSoftAssert.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- valiadte -->
</testsuite> <!-- validate Test -->
